app/views/pages/about.html.erb

```diff
@@ -0,0 +1,27 @@
+
+  <p>Initially released on 28 April 2016, this catalog is the result of a nearly two-year-long project to replace our twenty-year-old <%= t('blacklight.voyager') %> and incorporate the best features of contemporary library search technology.</p>
+
+  <p>The catalog is based on <a href="http://projectblacklight.org/" target="_blank">Blacklight</a>, which is an open source collaboration across several institutions including
+    <%= link_to("https://newcatalog.library.cornell.edu/", target: "_blank") { "Cornell" }  %>,
+    <%= link_to("https://catalyst.library.jhu.edu/", target: "_blank") { "Johns Hopkins" }  %>,
+    <%= link_to("https://searchworks.stanford.edu/", target: "_blank") { "Stanford" }  %>,
+    <%= link_to("http://search.library.yale.edu/", target: "_blank") { "Yale" }  %>, and
+    <%= link_to("http://projectblacklight.org/#examples", target: "_blank") { "a number of others" } %>.
+
+  <p>As the examples above demonstrate, Blacklight is highly configurable and customizable, which provides a number of advantages:</p>
+
+  <ul>
+    <li><strong>Tighter Integration with Existing Services.</strong> We run less risk of having to rely on third parties for integration with the several different requesting and fulfillment options that the library has to offer.</li>
+    <li><strong>Control over Look and Feel.</strong> We have complete contol over page layouts and how all aspects of the application are rendered. This allows us to create a more seamless experience for users as they pass between the catalog and library website, as well as ensure that the catalog is accessible and usable on mobile devices.</li>
+    <li><strong>Custom Relevancy.</strong> We know our collections and users better than anyone else, and the search engine beneath Blacklight gives us the power to tune search relevancy to best suit our environment.</li>
+    <li><strong>Community Development.</strong> As open source software developed by the academic and cultural heritage community, Blacklight allows us to leverage the talents of our own staff and dozens of software developers and librarians around the world.</li>
+    <li><strong>New Features and Fixes.</strong> The aforementioned community is always refining and adding new features to Blacklight. When a feature does not exist in the application or something is broken, we are able to build or fix it without having to wait for a vendor to do it for us.</li>
+  </ul>
+
+  <h2>Next Steps</h2>
+  <p>We will continue to refine the applicaiton over the summer by conducting a usability study, tuning relevance as we acquire usage statistics, finishing up a few key features, and ensuring that the application is performant and stable enough to handle typical user traffic.</p>
+  <p>We will update this page as we near our first official release, and in the meantime, we need your feedback! Please <a href="http://library.princeton.edu/help/contact-us" target="_blank">send us any questions or comments</a>.</p>
```

Comment:
'but in the meantime' instead of 'and in the meantime'. Also, is it okay that the initial release mentioned here is different from the initial release mentioned in notes.html.erb?